README.md

```diff
@@ -6,14 +6,14 @@ BigBlueButton supports real-time sharing of audio, video, slides (with whiteboar
 
 Presenters can record and playback content for later sharing with others.
 
-We designed BigBlueButton for online learning (though it can be used for many [other applications](http://www.c4isrnet.com/story/military-tech/disa/2015/02/11/disa-to-save-12m-defense-collaboration-services/23238997/)).  The educational use cases for BigBlueButton are
+We designed BigBlueButton for online learning, though it can be used for many other applications as well.  The educational use cases for BigBlueButton are
```

Comment:
```suggestion
We designed BigBlueButton for online learning, (though it can be used for many [other applications](https://www.c4isrnet.com/it-networks/2015/02/11/disa-to-replace-dco-with-new-collaboration-services-tool/) as well).  The educational use cases for BigBlueButton are
```

Thank you for spotting the dead links! This one seems to have changed slightly at the source website but the article is still up and is still as relevant.

Reply:
Thank you!